ynput/OpenPype
# Bugfix: houdini tab menu working as expected

## Changelog Description
This PR: 
1. Tab menu name changes to Ayon when using ayon
    ![image](https://github.com/ynput/OpenPype/assets/20871534/95822bb7-3f48-4975-b206-053e7c0afd78)

2. get_network_categories is checked in all creator plugins. 
    | Product | Network Category | 
    | -- | -- | 
    | Alembic camera | rop, obj |
    | Arnold Ass | rop |
    | Arnold ROP | rop |
    | Bgeo | rop, sop |
    | composite sequence | cop2, rop |
    | hda | obj |
    | Karma ROP | rop |
    | Mantra ROP | rop |
    | ABC | rop, sop |
    | RS proxy | rop, sop| 
    | RS ROP | rop |
    | Review | rop |
    | Static mesh | rop, obj, sop |
    | USD | lop, rop |
    | USD Render | rop |
    | VDB | rop, obj, sop |
    | V Ray | rop |
    

## Testing notes:
1. Use tab menu and check if you can find all products relevant to the network category
2. Check Tab menu name, it should be OpenPype when using OpenPype and AYON when using Ayon
